.gitignore

```diff
@@ -28,3 +28,4 @@ trie/test_data
 /bin
 chaindata/
 runtime/polkadot_runtime*
+runtime/test_wasm*
```

Comment:
Maybe we can replace these with `*.wasm`? That should stop any `.wasm` from ever being committed

Reply:
I want to ignore the whole folder

Reply:
What folder? These are referencing `runtime/polkadot_runtime.compact.wasm` and `runtime/test_wasm.wasm`, no?

Reply:
i have the test_wasm repo inside runtime/ so i wanted to ignore all of that, i can change it though